internal/hcsoci/create.go

```diff
@@ -54,6 +55,9 @@ type CreateOptions struct {
 	// ScaleCPULimitsToSandbox indicates that the container CPU limits should be adjusted to account
 	// for the difference in CPU count between the host and the UVM.
 	ScaleCPULimitsToSandbox bool
+
+	// disable creating containers with GMSA credentials (Spec.Windows.CredentialSpec)
+	NoGMSA bool
```

Comment:
This feels out of place on CreateOptions. There's no wcow/lcow specific configuration as of now. Granted I guess the alternative is to just parse the spec to see if that GMSA annotation is set on line 179 in this file, hmm

Reply:
That also felt out of place, since nothing else in `validateContainerConfig` checked annotations.
I am ambivalent, but I can see why having a creating option just for WCOW would be also be weird.
Maybe I can make it generic to `NoCredentials`, even though LCOW has no credentials (currently...)

Reply:
I'd honestly be fine with just parsing for the annotation on 179. If we ever do have a similar mechanism where introducing `NoCredentials` makes sense I'm fine with going with that afterwards